app/Http/Controllers/LocationsController.php

```diff
@@ -210,5 +210,29 @@ public function show($id = null)
 
         return redirect()->route('locations.index')->with('error', trans('admin/locations/message.does_not_exist'));
     }
+    
+public function print_assigned($id)
+    {
+
+        $location = Location::where('id',$id)->first();
+        $parent = Location::where('id',$location->parent_id)->first();
+	$manager = User::where('id',$location->manager_id)->first();
+	$users = User::where('location_id', $id)->with('company', 'department', 'location')->get();
+        $assets = Asset::where('assigned_to', $id)->where('assigned_type', Location::class)->with('model', 'model.category')->get();
```

Comment:
This looks pretty similar to a call below (in `print_all_assigned`) but with a few differences. Should there be two separate calls? I think *this* one is right, because `Assets` do have `assigned_to`, and that value is a Location ID if the `assigned_type` is a Location.